.github/workflows/docs.yml

```diff
@@ -18,10 +18,14 @@ concurrency:
 
 jobs:
   build:
+    env:
+      GITHUB_PAGES_URL: https://czbiohub.github.io/iohub
```

Comment:
this is not a huge deal but it would be more maintainable (particularly in light of our looming org rename) to construct this URL from the `GITHUB_REPOSITORY` env variable which afaik is defined by default in github actions. something like this:

```bash
REPO_OWNER="${GITHUB_REPOSITORY%%/*}";
REPO_NAME="${GITHUB_REPOSITORY#*/}";
GITHUB_PAGES_URL="https://$REPO_OWNER.github.io/$REPO_NAME";
```

Reply:
See https://github.com/czbiohub/iohub/pull/134/commits/1f3de567856cc534101afd8b2d7f581b03f6d082

Reply:
Nice!